checker/src/main/java/org/checkerframework/checker/initialization/InitializationChecker.java

```diff
@@ -87,6 +88,16 @@ public InitializationChecker() {}
      */
     public abstract Class<? extends BaseTypeChecker> getTargetCheckerClass();
 
+    /** Also handle {@code AnnotatedFor} annotations for the {@link InitializationChecker}. */
```

Comment:
I don't see an easy way to extract the common code, but maybe there is.

Definitely add links to the [InitializationFieldAccessSubchecker](https://github.com/eisop/checker-framework/blob/175f3fa0c6c3d5cac134b8e49eec78cca21bc625/checker/src/main/java/org/checkerframework/checker/initialization/InitializationFieldAccessSubchecker.java#L30C45-L30C68) version of the same code and document that these two versions should be kept in sync.

Reply:
Done.